### ISSUE TITLE
What should we do next?

### ISSUE THREAD

**Issue by yehud** [leela-zero/leela-zero#2487]
Compared with the previous，the speed of lz progressing is greatly reduced.What should we do next?
       60b? 3200visits？maybe we could use training data of elf and minigo（Recently published）？smaller learning rate？
      From 226 to242,Lz has almost no change in the winning rate with v17 ,(800playouts,1200games)
       I want to know what is the next step of lz

**Comment by yehud** [comment]
the winrate is about 55%

**Comment by l1t1** [comment]
we can test every weights of https://github.com/leela-zero/leela-zero/issues/2486, and add a column in the webpage for that
